assets/src/components/Sidebar/NewThemeSettings.js

```diff
@@ -56,12 +81,17 @@ const useAppliedCssVariables = (serverThemes, currentTheme) => {
 export const NewThemeSettings = ({ onChange, currentTheme }) => {
   const [selectedTheme, setSelectedTheme] = useState(currentTheme);
   const {editPost} = useDispatch('core/editor');
-  const serverThemes = useServerThemes();
+  const allServerThemes = useServerThemes();
+  const serverThemes = Object.entries(allServerThemes).reduce((a, [k, t]) => {
+    if (refactoredThemes.includes(k)) {
```

Comment:
This could be another approach
`return (refactoredThemes.includes(k)) ? a : ({ ...a, [k]: t })`

Reply:
@GP-Dan-Tovbein I also moved the function out and made the params more readable.